### PR TITLE
EDU-3257: Updates Task-Queue metrics coverage to reduce confusion

### PR DESCRIPTION
**Pull request by fairlydurable**
- "Describe" returns info that isn't metrics
- METRIC ACCURACY warnings are unnecessary clutter and scary

Expert: @atihkin

### CODE REVIEW

File: docs/develop/worker-performance.mdx [hunk 1]
```diff
@@ -37,22 +37,7 @@ The Workflow Cache is created and shared between all the workers. It's designed
...
-With an accurate estimate of backlog Tasks, you can determine the optimal number of Workers to deploy.
-Balance your Worker count with the number of Tasks to achieve the best performance.
-This approach minimizes Task backlog saturation and reduces idle Workers.
-
-Task Queue metrics provide numerical insights into your Task Queue activity and backlog characteristics.
-Use these metrics to tune your production deployments.
-Evaluate your Worker loads and assess whether you need to scale up or reduce your Worker deployment.
-
-### Task Queue types {#task-queue-types}
+## Task Queue types {#task-queue-types}
```
Comment (atihkin) [inline docs/develop/worker-performance.mdx:40]: Looking at the side nav now, "Task Queue types" as a main level heading isn't making sense unfortunately. As a user, I care about the "Available Task Queue Information"... the fact that there are different "Task Queue types" helps my understanding but isn't useful in and of itself.

File: docs/develop/worker-performance.mdx [hunk 2]
```diff
@@ -141,13 +110,24 @@ TasksAddRate - TasksDispatchRate
 - Positive values of `X` indicate the backlog is growing by about `X` Tasks per second.
 - Negative values of `X` indicate the backlog is shrinking by about `X` Tasks per second.
 
-:::info Metric Accuracy
-
 While individual `add` and `dispatch` rates may be inaccurate due to Eager and Sticky Task Queues, the `BacklogIncreaseRate` reliably reflects the rate at which the backlog is shrinking or growing for backlogs older than a few seconds.
 
-:::
+## Evaluate Task Queue performance {#evaluate-worker-loads}
```
Comment (atihkin) [inline docs/develop/worker-performance.mdx:115]: Maybe swap "Evaluate" with "Monitor"?

File: docs/develop/worker-performance.mdx [hunk 3]
```diff
@@ -141,13 +110,24 @@ TasksAddRate - TasksDispatchRate
...
+With an accurate estimate of backlog Tasks, you can determine the optimal number of Workers to deploy.
+Balance your Worker count with the number of Tasks to achieve the best performance.
+This approach minimizes Task backlog saturation and reduces idle Workers.
+
+Task Queue data provide numerical insights into your Task Queue activity and backlog characteristics.
+Use these numbers to tune your production deployments.
+Evaluate your Worker loads and assess whether you need to scale up or reduce your Worker deployment.
 
-### Fetch Worker metrics {#retrieve-worker-info}
+### Query Task Queue info with Temporal CLI {#cli-task-queue-info}
```
Comment (atihkin) [inline docs/develop/worker-performance.mdx:130]: This is nice. I would like to add another section similar to this but which says "Query Task Queue info with the Go SDK" and have a little code snippet. Does that work? I can share the code snippet here.

File: docs/develop/worker-performance.mdx [hunk 4]
```diff
@@ -68,68 +53,52 @@ For each Task Queue name, Temporal creates separate queues for each Task Queue t
...
-The Temporal Service reports information separately for each Task Queue type (not aggregated).
-Use the following metrics to retrieve detailed information about Task Queue health and performance.
-Available metrics include:
+:::tip Support, stability, and dependency info
 
-- [`ApproximateBacklogCount`](#ApproximateBacklogCount)
-- [`ApproximateBacklogAge`](#ApproximateBacklogAge)
-- [`TasksAddRate`](#TasksAddRate-and-TasksDispatchRate) and [`TasksDispatchRate`](#TasksAddRate-and-TasksDispatchRate)
-- [`BacklogIncreaseRate`](#BacklogIncreaseRate) (derived from [`TasksAddRate`](#TasksAddRate-and-TasksDispatchRate) and [`TasksDispatchRate`](#TasksAddRate-and-TasksDispatchRate))
+The information listed in this section is currently readable using the Go SDK, Temporal CLI, and direct [RPC](https://github.com/temporalio/temporal/blob/c4b6b34f33f4772564c8ae759f2e907e3651a65a/service/matching/matching_engine.go) calls. (See [`DescribeTaskQueue`](https://github.com/search?q=repo%3Atemporalio%2Ftemporal%20DescribeTaskQueue&type=code).)
```
[the remark this answers is not in the document]
Comment (fairlydurable) [reply docs/develop/worker-performance.mdx]: Revised during our sync.

File: docs/develop/worker-performance.mdx (hunk 1 above)
Comment (fairlydurable) [reply docs/develop/worker-performance.mdx]: Revised per suggestion.

File: docs/develop/worker-performance.mdx (hunk 2 above)
Comment (fairlydurable) [reply docs/develop/worker-performance.mdx:97]: Resolving after sync

File: docs/develop/worker-performance.mdx (hunk 3 above)
Comment (fairlydurable) [reply docs/develop/worker-performance.mdx:118]: RPS limits added, paralleled  in Go section, during our sync

File: docs/develop/worker-performance.mdx [hunk 5]
```diff
@@ -168,7 +137,27 @@ This feature may significantly change or be removed in a future release.
 
 :::
 
-#### Evaluate Worker availability and capacity issues {#worker-capacity-issues}
+### Query Task Queue info with Go SDK {#go-sdk-task-queue-info}
```
Comment (napcs) [inline docs/develop/worker-performance.mdx:140]:
```suggestion
### Query Task Queue info with the Go SDK {#go-sdk-task-queue-info}
```

File: docs/develop/worker-performance.mdx [hunk 6]
```diff
@@ -168,7 +137,27 @@ This feature may significantly change or be removed in a future release.
...
+
+Retrieve Task Queue data using the Go SDK by calling `DescribeTaskQueueEnhanced`.
+Specify the Task Queue name and set `ReportStats` to `true`, as in the following example:
+
+```go
```
Comment (napcs) [inline docs/develop/worker-performance.mdx:145]: snipsync?

**Review by napcs — APPROVED** [review]

**Pull request merged**